components/OrderStatusTag.js

```diff
@@ -33,7 +33,7 @@ const msg = defineMessages({
     defaultMessage: 'cancelled',
   },
   [ORDER_STATUS.REJECTED]: {
-    id: 'Order.Status.Rejected',
```

Comment:
Here we need to keep differentiating between a rejected expense and a rejected order